monitor/group_vars/all.yml

```diff
@@ -72,18 +72,18 @@ sensu:
       password: sensu
       vhost: /sensu
       hostgroup: sensu
-#    handlers:
-#      default:
-#        - flapjack_http
-#      flapjack:
-#        enabled: False
-#        host: 127.0.0.1
-#        port: 6380
-#      flapjack_http:
-#        enabled: True
-#        uri: http://127.0.0.1:3090
-#        ttl: -1 # -1 disables, otherwise httpbroker re-sends last received state until ttl expires
-#        service_owner: development
+  # handlers:
```

Comment:
we can nuke these as well, we don't need to flapjack for this demo so probably wouldn't uncomment them back in.